src/components/TextInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Field, propTypes } from 'redux-form';
+import { TextField } from 'redux-form-material-ui';
+
+
+const TextInput = ({ name, type, placeholder, ...props }) => (
+  <Field
+    name={name}
+    component={TextField}
+    type={type}
+    {...props}
+    placeholder={placeholder}
+  />
+  );
+TextInput.propTypes = {
```

Comment:
I added all three properties (name, type, placeholder) unless, placeholder and type are not required